packages/clients/textLocator/src/plugins/GeometrySearch/store/actions/setupTooltip.ts

```diff
@@ -0,0 +1,50 @@
+import { Tooltip, getTooltip } from '@polar/lib-tooltip'
+import { Feature, Overlay } from 'ol'
+import { vectorLayer } from '../../utils/vectorDisplay'
+import { getPrimaryName } from '../../../../utils/coastalGazetteer/getPrimaryName'
+
+const localeKeys: [string, string][] = [
+  ['h2', 'plugins.geometrySearch.tooltip.title'],
+]
+
+export function setupTooltip({ rootGetters: { map } }) {
```

Comment:
This function seems like a duplicate to the `useExtendedMasterportalapiMarkers` of `@polar/core`.

Have you tried making the core implementation work with the local vectorLayer?

Reply:
Those methods do not look alike to me.

Reply:
The functions itself do not look similar, ofc.  
They both add a tooltip (hovered) to the map.

I assume you didn't try and we can leave it be, if you want 🎩